tasks/lint-all.js

```diff
@@ -4,11 +4,118 @@ var gpii = fluid.registerNamespace("gpii");
 
 require("../index");
 require("../src/load-npm-tasks-properly");
+require("gpii-glob");
 
 module.exports = function (grunt) {
     gpii.grunt.lintAll.fixGruntTaskLoading(grunt);
 
-    grunt.config.merge({
+    gpii.grunt.lintAll.mergeAndExpandOptions = function (grunt) {
+        var sanelyMergedOptions = {};
+        fluid.each(
+            ["lintAll", "eslint", "jsonlint", "json5lint", "markdownlint", "mdjsonlint", "json-eslint", "lintspaces"],
```

Comment:
This array is just the one on line 110 with an extra first element. Spare the reader slightly by putting the core of it into a constant or config option

Reply:
There were actually a couple more uses scattered throughout, I standardised all variations so that it'll be easier to add new tasks in the future.